bin/build-pxp-agent.ps1

```diff
@@ -33,7 +33,7 @@ $cmake_args = @(
   "-DBOOST_STATIC=ON",
   "-DYAMLCPP_ROOT=`"$toolsDir\$yamlPkg`"",
   "-DCMAKE_INSTALL_PREFIX=`"$sourceDir`"",
-  "-DCMAKE_PREFIX_PATH=`"$toolsDir\$curlPkg;$toolsDir\pcp-client`"",
+  "-DCMAKE_PREFIX_PATH=`"$toolsDir\$curlPkg;$toolsDir\$opensslPkg;$toolsDir\$rubyPkg;$toolsDir\pcp-client`"",
   "-DCURL_STATIC=ON",
   ".."
```

Comment:
`cpplint` is run in Travis CI; I don't see much reason to running it again as part of packaging, so Python isn't needed.